### PR TITLE
Presearch: Add left out changes & fix better targeted ones

### PR DESCRIPTION
**Pull request by karolyi**
Reiterating previously left out changes and fixing ones that do nothing or target ads less efficiently:

1: `presearch.com###background-cover` does nothing, by my tests
2: `presearch.com##.items-center.bg-transparent` is better targeted by `presearch.com##[href*="background-url"]`
3: Block autosuggestions that often contain ads
4: Block carousels that are filled with links leading to bigtech (mostly amazon) via click tracking links.

### CODE REVIEW

**Comment by karolyi** [comment]
Bump for unfucking this increasingly cringe search engine, for the time I keep using it

**Comment by ryanbr** [comment]
I see no reason to merge this, the current filters are doing its job. I see no ads

**Comment by karolyi** [comment]
This is a typical case of _works on my machine™_. What proof do you need that it successfully blocks ads and sponsors, and basically everything I wrote in the initial comment?

Presearch is a decentralized search engine, nodes with various versions serving the results. I'm currently getting "Results from [decentralized nodes](https://network.presearch.com/) in the eu-de-1 region." I already explained [in the other issue](https://github.com/easylist/easylist/pull/17863), what nodes/requests should be blocked and why. I'm getting out of options here.

My ruleset might be of invalid syntax, because I use ublock origin. I already made efforts to dumb it down to ABP and other levels, might be it wasn't enough. Help me dumb it down further and I'll try and do my best to get them merged.

**Comment by karolyi** [comment]
The autocomplete issue with the ads is still not fixed. I'll keep this ticket as a reference with the proper blocking results.

Anyone who's as annoyed by it as me, just copy the rules I provided into your own custom ruleset. They work flawlessly for me.
